lib/initializer.rb

```diff
@@ -0,0 +1,56 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+
+class Initializer
+  attr_accessor :logic1
+
+  def initialize(size, empty_char)
+    @size = size
+    @empty_char = empty_char
+    @player1
+    @player2
```

Comment:
null instance variables are a code smell to me, is it possible that we could inject the players instead?